functions-methods.js

```diff
@@ -0,0 +1,61 @@
+// Je gaat functies schrijven die we kunnen hergebruiken om sommige emailadressen te checken. Nu zul je gaan merken hoe handig functies kunnen zijn!
+// Je zult hier methoden van het String Object voor nodig hebben, dus pak de paragraaf op EdHub over het String Object er even bij.
+
+
+/* Opdracht  1 */
+// Schrijf een functie genaamd getEmailDomain, die een emailadres verwacht en de domeinnaam teruggeeft. Een domeinnaam is hetgeen dat na het @ in het adres staat
+// ---- Verwachte uitkomsten:
+// getEmailDomain("n.eeken@novi-education.nl") geeft novi-education.nl
+// getEmailDomain("t.mellink@novi.nl") geeft novi.nl
+// getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
+
+const getEmailDomain = (email) => email.slice((email.indexOf("@")+1), email.length)
```

Comment:
Je gebruikt 'function naam-van-functie()' en 'const naam-van-functie = () =>' door elkaar heen. Beide zijn goed, maar het is netter om 1 van de twee te gebruiken en niet allebei omstebeurt.